sdk/storage/pom.service.xml

```diff
@@ -9,7 +9,7 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
-<!--   <module>microsoft-azure-storage-blob</module>-->
+   <module>microsoft-azure-storage-blob</module>
```

Comment:
Why did we enable the track 1 library to run CI during check in?

Reply:
Comment out.